src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Ball ball = new Ball();
+        Random random = new Random();
+        ColorSupplier colorSupplier = new ColorSupplier();
+
+        ball.setNumber(random.nextInt(100));
```

Comment:
bound parameter of `nextint()` is exclusive which means that max number we can get right now is 99, we should fix that and make that number a constant